mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
+msgstr "Paso dentro"
 
-#: mu/modes/debugger.py:67
+#: mu/modes/debugger.py:72
 msgid "Step into a function."
-msgstr "STEP INTO A FUNCTION."
+msgstr "Entrar en una función."
 
-#: mu/modes/debugger.py:72
+#: mu/modes/debugger.py:78
 msgid "Step Out"
-msgstr "STEP OUT"
+msgstr "Paso fuera"
 
-#: mu/modes/debugger.py:73
+#: mu/modes/debugger.py:79
 msgid "Step out of a function."
-msgstr "STEP OUT OF A FUNCTION."
+msgstr "Salir de una función."
 
-#: mu/modes/debugger.py:110
-msgid "Running script {}"
-msgstr "RUNNING SCRIPT {}"
-
-#: mu/modes/debugger.py:149
+#: mu/modes/debugger.py:156
 msgid "Your script has finished running."
-msgstr "YOUR SCRIPT HAS FINISHED RUNNING."
+msgstr "Tu script ha terminado de ejecutarse."
+
+#: mu/modes/debugger.py:224
+msgid ""
+"Unable to connect to the Python debugger.\n"
+"\n"
+msgstr ""
+"No ha sido posible conectar con el depurador de Python.\n"
+"\n"
 
-#: mu/modes/debugger.py:270
+#: mu/modes/debugger.py:308
 msgid "Debugger info: {}"
-msgstr "DEBUGGER INFO: {}"
+msgstr "Información del depurador: {}"
 
-#: mu/modes/debugger.py:276
+#: mu/modes/debugger.py:314
 msgid "Debugger warning: {}"
-msgstr "DEBUGGER WARNING: {}"
+msgstr "Advertencia del depurador: {}"
 
-#: mu/modes/debugger.py:283
+#: mu/modes/debugger.py:321
 msgid "Debugger error: {}"
-msgstr "DEBUGGER ERROR: {}"
+msgstr "Error del depurador: {}"
 
-#: mu/modes/microbit.py:34
+#: mu/modes/microbit.py:168
 msgid "BBC micro:bit"
-msgstr "BBC MICRO:BIT"
+msgstr "BBC micro:bit"
 
-#: mu/modes/microbit.py:35
+#: mu/modes/microbit.py:169
 msgid "Write MicroPython for the BBC micro:bit."
-msgstr "WRITE MICROPYTHON FOR THE BBC MICRO:BIT"
+msgstr "Escribe MicroPython para el BBC micro:bit."
 
-#: mu/modes/microbit.py:53
+#: mu/modes/microbit.py:190
+#, fuzzy
 msgid "Flash"
-msgstr "FLASH"
+msgstr "Flash"
 
-#: mu/modes/microbit.py:54
+#: mu/modes/microbit.py:191
 msgid "Flash your code onto the micro:bit."
-msgstr "FLASH YOUR CODE ONTO THE MICRO:BIT"
+msgstr "Programa tu código en el micro:bit."
 
-#: mu/modes/microbit.py:59
+#: mu/modes/microbit.py:197
 msgid "Files"
-msgstr "FILES"
+msgstr "Archivos"
 
-#: mu/modes/microbit.py:60
+#: mu/modes/microbit.py:198
 msgid "Access the file system on the micro:bit."
-msgstr "ACCESS THE FILE SYSTEM ON THE MICRO:BIT"
+msgstr "Accede al sistema de archivos en el micro:bit."
 
-#: mu/modes/microbit.py:66
+#: mu/modes/microbit.py:204 mu/modes/python3.py:127
+msgid "REPL"
+msgstr "REPL"
+
+#: mu/modes/microbit.py:205
 msgid "Use the REPL to live-code on the micro:bit."
-msgstr "USE THE REPL TO LIVE-CODE ON THE MICRO:BIT"
+msgstr "Use el REPL para programar en vivo en el micro:bit."
 
-#: mu/modes/microbit.py:94
+#: mu/modes/microbit.py:255
 msgid "Unable to flash \"{}\""
-msgstr "UNABLE TO FLASH \"{}\""
+msgstr "No se puede programar \"{}\""
+
+#: mu/modes/microbit.py:265
+msgid "Problem with script"
+msgstr "Problema con el script"
+
+#: mu/modes/microbit.py:266
+msgid "{} [{}:{}]"
+msgstr "{} [{}: {}]"
```

Comment:
Perhaps this one should be removed as a translation?

Reply:
OK... so, I'm assuming people translating also look at the code context. Here's why it's translateable:

* This is a string that gets shown to the end user (and all such strings should be translateable).
* The items to be interpolated are English error messages from Python. The translator may want to add context here, so they make more sense to a non-English speaker.

Does this make sense?

Reply:
Yes, definitely makes sense, thanks!